app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :password, presence: true, length: {minimum: 6}
```

Comment:
Các giá trị như 6, 255m 50 thì em khai báo vào file Settings

Để làm được điều này, em tìm hiểu `gem config in rails ` nhé